components/AntHiveGameVertical.vue

```diff
@@ -51,7 +51,7 @@
             </span>
           </div>
           <div>
-            <span>{{ $t("game.rt") }}:</span>
```

Comment:
This is correct the vertical Bot has ART - average response time  (calculated from all ticks from all games)